test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -287,29 +480,41 @@ test.requestHooks(enrollViaEmailMocks)('should be able enroll via email', async
   await t.expect(enrollViaSMSPageObject.hasCountryField()).notOk();
   await t.expect(enrollViaEmailPageObject.hasSwitchChannelText).ok();
   await enrollViaEmailPageObject.fillEmailField('test@gmail.com');
-  await enrollViaEmailPageObject.clickNextButton();
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(false);
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(true);
-  await t.expect(enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(false);
-  await t.expect(enrollOktaVerifyPage.getEmailInstruction()).eql(emailInstruction);
+  await enrollViaEmailPageObject.clickSendSetupLink();
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaQRInstruction()).eql(false);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaEmailInstruction()).eql(true);
+  await t.expect(await enrollOktaVerifyPage.hasEnrollViaSmsInstruction()).eql(false);
+  await t.expect(enrollOktaVerifyPage.getEmailInstruction()).contains(emailInstruction1);
+  if (!userVariables.v3) {
+    await t.expect(enrollOktaVerifyPage.getEmailInstruction()).contains(emailInstruction2);
+  }
+  
+  isSuccess = true;
   const successPage = new SuccessPageObject(t);
   const pageUrl = await successPage.getPageUrl();
   await t.expect(pageUrl)
     .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
+  isSuccess = false;
 });
 
 test.requestHooks(resendEmailMocks)('after timeout should be able see and click send again link when enrolling via email', async t => {
   const enrollOktaVerifyPage = await setup(t);
   await checkA11y(t);
-  await t.expect(enrollOktaVerifyPage.getEmailInstruction()).eql(emailInstruction);
-  await t.expect(enrollOktaVerifyPage.resendView().visible).notOk();
+  await t.expect(enrollOktaVerifyPage.getEmailInstruction()).contains(emailInstruction1);
+  if (!userVariables.v3) {
+    await t.expect(enrollOktaVerifyPage.getEmailInstruction()).contains(emailInstruction2);
+  }
```

Comment:
In v3, the 2nd part of the instruction `Or try a different way to set up Okta Verify.` is rendered in a separate div and anchor tag. Couldn't find an intelligent way to extract since there's no unique selector. Let me see if I can find some work around
<img width="1054" alt="Screenshot 2023-04-19 at 2 49 13 PM" src="https://user-images.githubusercontent.com/27521424/233207817-ae36f03e-3745-4349-b878-8739fe132285.png">

Reply:
A workaround would be to grab the entire text using `[data-se="o-form-explain"]` and verify the message exists.
<img width="1000" alt="Screenshot 2023-04-19 at 2 53 24 PM" src="https://user-images.githubusercontent.com/27521424/233208325-5ed230b2-283b-4aea-9f6f-84d2954a1f80.png">

Reply:
Pushed this change in a new commit. Removed the conditional check.